app/src/androidTest/java/org/kiwix/kiwixmobile/localFileTransfer/LocalFileTransferRobot.kt

```diff
@@ -0,0 +1,37 @@
+/*
+ * Kiwix Android
+ * Copyright (c) 2020 Kiwix <android.kiwix.org>
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package org.kiwix.kiwixmobile.localFileTransfer
+
+import applyWithViewHierarchyPrinting
+import org.kiwix.kiwixmobile.BaseRobot
+import org.kiwix.kiwixmobile.Findable
+
+/**
+ * Authored by Ayush Shrivastava on 29/10/20
+ */
+
+fun localFileTransfer(func: LocalFileTransferRobot.() -> Unit) =
+  LocalFileTransferRobot().applyWithViewHierarchyPrinting(func)
+
+class LocalFileTransferRobot : BaseRobot() {
+
+  init {
+    isVisible(Findable.Text("Receive Files"))
```

Comment:
Prefer TextId and import Findable so you can just write `TextId(R.string.something)`

Reply:
Actually, I couldn't find this text in the strings file so that's why I've hardcoded it here. So creating it in the concerned file is necessary?

Reply:
it is there `R.string.receive_files_title`

Reply:
![ss](https://user-images.githubusercontent.com/56556919/98078658-7d3dda80-1e98-11eb-9cda-5642c2aa2127.png)

Reply:
![image](https://user-images.githubusercontent.com/3358282/98091987-10b5e200-1e7e-11eb-958a-39ec330bf409.png)
this is a multimodular project, our strings are not centralised in one location